Library/Homebrew/utils/shared_audits.rb

```diff
@@ -46,6 +46,7 @@ def github_release_data(user, repo, tag)
     "telegram-cli"     => "1.3.1",
     "toggl-track"      => :all,
     "volta"            => "0.8.6",
+    "gron"             => :all,
```

Comment:
Sorry to be picky, but would you mind putting this in alphabetical order? I regularly add Casks to this list, and having them in alphabetical order really helps. Thanks.

Reply:
Sure, no problem!

Reply:
Thank you @Moisan.